project/src/test/java/com/google/EventTest.java

```diff
@@ -0,0 +1,94 @@
+package com.google.sps;
+
+import com.google.sps.data.Event;
+import com.google.sps.data.User;
+import com.google.sps.data.VolunteeringOpportunity;
+import java.time.LocalDate;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class EventTest {
+    private static final String HOST_NAME = "Bob Smith";
+    private static final String EMAIL = "bobsmith@google.com";
+    private static final User HOST = new User.Builder(HOST_NAME, EMAIL).build();
+    private static final String EVENT_NAME = "Team Meeting";
+    private static final String NEW_EVENT_NAME = "Daily Team Meeting";
+    private static final String DESCRIPTION = "Daily Team Sync";
+    private static final Set <String> LABELS = Collections.unmodifiableSet(new HashSet<>(Arrays.asList("Tech", "Work")));
+    private static final String LOCATION = "Remote";
+    private static final LocalDate DATE = LocalDate.of(2020, 7, 24);
+    private static final String VOLUNTEER_OPPORTUNITY_NAME = "Volunteer";
+    private static final int NUM_SPOTS_LEFT = 5;
+    private static final VolunteeringOpportunity OPPORTUNITY = new VolunteeringOpportunity.Builder(VOLUNTEER_OPPORTUNITY_NAME, NUM_SPOTS_LEFT).build();
+    private static final Set <VolunteeringOpportunity> OPPORTUNITIES = new HashSet<VolunteeringOpportunity> (Arrays.asList(OPPORTUNITY));
```

Comment:
Maybe you can use ImmutableSet.of instead of Arrays.asList?

Reply:
the set is mutated by the 3rd test (adding / removing from the set)